src/css/fonts.css

```diff
@@ -2,4 +2,12 @@
   --font-primary: normal normal 600 normal 16px/1.5 Poppins, sans-serif;
```

Comment:
Would it work if we replaced this "16px" too with the `--font-size-2base` variable? 🤔

Reply:
think this would work @larimaza 

```suggestion
  --font-primary: normal normal 600 normal var(--font-size-2base)/1.5 Poppins, sans-serif;
```

Reply:
@larimaza Ah, nice catch! Will do that ASAP!

Reply:
@larimaza @luisfmsouza Done!